axiom/nr_distributed_trace.c

```diff
@@ -489,8 +489,8 @@ void nr_distributed_trace_set_trace_id(nr_distributed_trace_t* dt,
   nr_free(dt->trace_id);
   if (trace_id) {
     int len = nr_strlen(trace_id);
-    if (len < 33) {
-      int padding = 33 - len;
+    if (len < 32) {
```

Comment:
It'd probably be helpful to make this value a DEFINE.

Reply:
done in [c1f11ed](https://github.com/newrelic/newrelic-php-agent/pull/929/commits/c1f11edf870a58b7f87e93c2b0be908b2099f1d5)

Reply:
After rebase the commit id is 68d8d5c6aedcb990d4596f00e412b1cceaa24fdc.